config/yamlcfg/yaml2proto_test.go

```diff
@@ -499,7 +521,10 @@ test_groups:
 			input: config.Configuration{
 				TestGroups: []*config.TestGroup{
 					{
-						Name: "test_group",
+						Name:             "test_group",
+						GcsPrefix:        "fake path",
```

Comment:
FYI: I'm not super wild about having so many fields unrelated to "basic dashboard values work" as it makes these cases harder to understand what we're actually trying to test (not GcsPrefix for sure). I don't have a great solution though off-hand

Reply:
Yeeeeeah, I agree. I was thinking of breaking out a default TestGroup to add to the tests, but I'm not sure if the tradeoff on explicitness is worth it here.

Reply:
Changed, might be better?